src/global.ts

```diff
@@ -144,6 +144,11 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
    * Changes to recommended defaults, like defaultScope
    */
   advancedOptions?: AdvancedOptions;
+
+  /**
+   * Number of days until the cookie `auth0.is.authenticated` will expire
+   */
+  daysUntilAuthenticateCookieExpire?: number;
```

Comment:
I think the name should reflect what the property is changing from an API point of view. Yes, it's affecting the expiry of the underlying cookie, but that's an internal detail. From an outsider's perspective, this is actually changing the behaviour of `checkSession`.

Maybe something like `sessionCheckExpiryDays`.

Reply:
Good point of view, I agree we should name considering it. I dedicated some time but I couldn't came up with a better name, so I followed your suggestion.